docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -551,36 +549,44 @@ Multiple server's abilities can be set to give information to the client about s
     - displayRotation - Abilities of setting display orientations.
     - displayMode - Abilities of setting display modes.
 
-To set abilities on server side you can use `saveAbilities()` method.
+### Setting abilities
 
 ```
-server.updatePlaybackState("PLAY");
-server.abilities.playback.next = "YES";
-server.abilities.playback.prev = "YES";
-server.abilities.playback.rewind = "NO";
-server.abilities.playback.forward = "NO";
+mcServer.updatePlaybackState("PLAY");
```

Comment:
i think it can be deleted. On server side you don't have to update state. It is only for updating playback info.

Reply:
Done.